service_mesh/service_mesh_day_two/ossm-auto-route.adoc

```diff
@@ -0,0 +1,69 @@
+[id="ossm-auto-route"]
+= Automatic route creation
+include::modules/ossm-document-attributes.adoc[]
+toc::[]
+
+{product-title} routes for Istio Gateways are automatically managed in {ProductName}. Every time an Istio Gateway is created, updated or deleted inside the service mesh, an {product-title} route is created, updated or deleted. 
+
+[id="ossm-auto-route-enable_{context}"]
+== Enabling Automatic Route Creation
+A {ProductName} control plane component called Istio {product-title} Routing (IOR) synchronizes the gateway route. You can deploy it as part of the control plane in the {ProductName} installation process, but you must enable it first.
+
+. In the `ServiceMeshControlPlane` resource, add the `ior_enabled` parameter and set it to `true`. For example, see the following resource snippet:
+
+----
+gateways:
+ istio-egressgateway:
+   autoscaleEnabled: false
+   autoscaleMin: 1
+   autoscaleMax: 5
+ istio-ingressgateway:
+   autoscaleEnabled: false
+   autoscaleMin: 1
+   autoscaleMax: 5
+   ior_enabled: true
+----
+
+For more information, see xref:../../service_mesh/service_mesh_install/customizing-installation-ossm.adoc#ossm-cr-gateway_customizing-installation-ossm[Istio gateway configuration].
+
+[id="ossm-auto-route-subdomains_{context}"]
+== Subdomains
+
+{ProductName} creates the route with the subdomain, but {product-title} must be configured to enable it. Subdomains, for example, `*.domain.com` are supported but not by default. Cluster administrators can refer to the {product-title} documentation for instructions on how to enable subdomains.
```

Comment:
"Cluster administrators can refer to the {product-title} documentation for instructions on how to enable subdomains."
Have you found the docs yet?  It would be nice if we could provide the link.

Reply:
No, I'm working on it.